RocketBot2/Config/Translations/translation.ca.json

```diff
@@ -252,11 +252,11 @@
     },
     {
       "Key": "logEntryLevelUP",
-      "Value": "Level Up"
+      "Value": "LEVEL UP"
```

Comment:
LEVEL UP is an value Key as good value it is translation CA (canada) it as upercase because POKEMON) GYM) EVOLE) etc is upercase also and it is copy ans pas of Necrobot.Cli/Config an sync translate new in rocket is not one edition